x-pack/test/api_integration/deployment_agnostic/apis/observability/streams/lifecycle.ts

```diff
@@ -84,6 +85,7 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
 
     const wiredPutBody: IngestStreamUpsertRequest = {
       stream: {
+        description: 'irrelevant',
```

Comment:
should these be in the lifecycle tests? or maybe we should rename to update?

Reply:
I was looking for some usage of putStream, but probably not the right place - Where would you move them?

Reply:
I'm fine with something like `update_stream.ts` or `put_stream.ts`. I'm also fine with `description.ts` although that seems overly specific.